src/delivery-options/data/pickup/sortPickupLocations.js

```diff
@@ -17,6 +17,6 @@ export function sortPickupLocations(pickupLocations) {
     const comparisonA = locationA[sortKey];
     const comparisonB = locationB[sortKey];
 
-    return comparisonA < comparisonB ? -1 : 1;
+    return comparisonA - comparisonB;
```

Comment:
Als ik het goed lees kan `comparison...` zowel distance (number) als location name (string) bevatten. Een string minus een string is altijd NaN wat voor nog veel funkyer sorteervolgorde zorgt

Reply:
Ik heb het gefixt en tests toegevoegd 👍